trainingMaquetado/maquetado.css

```diff
@@ -1,3 +1,23 @@
+.paragraph {
+  color: #999;
+  font-family: "Open Sans";
+  font-size: 15px;
+  font-weight: bold;
+  padding: 10px 0;
+  text-align: center;
+  width: 190px;
+}
+
+.title {
+  color: #111;
+  font-family: Open Sans;
```

Comment:
this should be wrapped in "" . To avoid repeating this everywhere, since you'll always use the same font you can do this:
```
* {
  font-family: "Open Sans", sans-serif;
}
```

that will asign the font to every element